packages/react/src/exchanges/hooks/useExchangeFilters.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  createExchangeFilter as createExchangeFilterAction,
+  getExchangeFilterError,
+  getExchangeFilters,
+  isExchangeFilterLoading,
+  resetExchangeFilters,
+  type StoreState,
+} from '@farfetch/blackout-redux';
+import { useCallback, useMemo } from 'react';
+import { useSelector } from 'react-redux';
+import useAction from '../../helpers/useAction.js';
+import type {
+  Config,
+  ExchangeFilterItem,
+  OrderItem,
+  PostExchangeFilterData,
+} from '@farfetch/blackout-client';
+
+type PostExchangeFilterDataHook = PostExchangeFilterData & {
```

Comment:
This type must be exported since they are part of a public method (This applies to `ExchangeFilterItemWithOptionalUuid` as well)

Reply:
Done

Reply:
This needs to be reexported at the index file. Please, check the file `/Users/bruno.a.oliveira/Code/blackout/packages/react/src/exchanges/hooks/types/useExchangeBookRequest.ts` and make the same changes for this file.